### PR TITLE
feat(networks): add support for Scroll network

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This PR adds support for Scroll network

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
need to rebase when #13249 is merged

File: packages/networks/src/networks/scroll.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const scroll: NetworkConfig = {
+  chain: 'ethereum',
```
Comment (julien51) [inline packages/networks/src/networks/scroll.ts:4]:
```suggestion
  chain: 'scroll',
```

File: packages/networks/src/networks/scroll.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  description:
+    'The original and most secure EVM network. Gas fees are expensive on this network.',
```
Comment (julien51) [inline packages/networks/src/networks/scroll.ts:6]:
```suggestion
  description:
    'Scroll seamlessly extends Ethereum’s capabilities through zero knowledge tech and EVM compatibility. The L2 network built by Ethereum devs for Ethereum devs.',
```

File: packages/networks/src/networks/scroll.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  explorer: {
+    name: 'Etherscan',
```
Comment (julien51) [inline packages/networks/src/networks/scroll.ts:8]:
```suggestion
    name: 'Scrollscan',
```

File: packages/networks/src/networks/scroll.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      base: 'https://scrollscan.com/',
+      token: (address, holder) =>
+        `https://etherscan.com/token/${address}?a=${holder}`,
+      transaction: (hash) => `https://scrollscan.com/tx/${hash}`,
+    },
+  },
+  featured: false,
+  id: 534352,
+  isTestNetwork: false,
+  multisig: '',
```
Comment (julien51) [inline packages/networks/src/networks/scroll.ts:20]: @clemsos deployed?

File: packages/networks/src/networks/scroll.ts [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  name: 'Scroll',
+  nativeCurrency: {
+    coingecko: 'ethereum',
+    decimals: 18,
+    name: 'Ether',
+    symbol: 'ETH',
+    wrapped: '0x5300000000000000000000000000000000000004',
+  },
+  provider: 'https://rpc.unlock-protocol.com/534352',
```
Comment (julien51) [inline packages/networks/src/networks/scroll.ts:29]: I probably need to deploy that.

**Review by julien51 — COMMENTED** [review]
A few questions/comments. Do you mind trying to split in 2 PRs plase? One with the just scroll changes and one with the otehr changes...

**Comment by clemsos** [comment]
> A few questions/comments. Do you mind trying to split in 2 PRs plase? One with the just scroll changes and one with the otehr changes...

Yes the second PR is there https://github.com/unlock-protocol/unlock/pull/13249

File: packages/networks/src/networks/scroll.ts (hunk 4 above)
Comment (clemsos) [reply packages/networks/src/networks/scroll.ts]: yes doen

File: governance/scripts/multisig/_helpers.js
Comment (clemsos) [inline governance/scripts/multisig/_helpers.js:1]: moved to #13294

File: packages/networks/src/networks/scroll.ts (hunk 5 above)
Comment (julien51) [reply packages/networks/src/networks/scroll.ts:29]: I have set the env vars and I will deploy now!

Comment (julien51) [reply packages/networks/src/networks/scroll.ts:29]: Env var set and deployment done!

**Comment by julien51** [comment]
We need to merge this one before I can deploy the RPC for zkEVM!

**Pull request merged**